src/components/modal/v2/parts/views/ShortTerm/styles.scss

```
@@ -105,6 +105,18 @@
         @include apiIframe {
             max-width: $content-wrapper-max-width-api-iframe;
         }
+
+        &.checkout {
+            .button {
+                @include tablet {
+                    width: auto;
+                }
+            }
+        }
+    }
+
+    .checkout {
+        background: $ql-gray;
```

```suggestion
        background: $ql-gray;
        
            .donuts__container {
            margin-top: -30px;
            padding-bottom: 0px;
            ::after {
                margin-top: -30px;
            }
        }
```
Since there's a bit of extra space on the donuts container, we can shift it up a bit here.